src/containers/ImportRecordingsInstructions.tsx

```diff
@@ -22,6 +23,10 @@ const ImportRecordingsInstructions: FunctionComponent<Props> = () => {
                 <CopyToClipboardButton text={import_spikeforest_recording_py} />
                 <Markdown source={mdWrapPy(import_spikeforest_recording_py)} />
             </Expandable>
+            <Expandable label="Import NWB recordings">
```

Comment:
Here's where we provide the instructions (example python snippet) for importing nwb recording/sorting